src/ui/events-list-view.cpp

```diff
@@ -100,16 +101,32 @@ void EventItemDelegate::paint(QPainter *painter,
         avatar = AvatarService::instance()->getAvatar(event.author);
     }
     if (avatar.size() != QSize(kAvatarWidth, kAvatarHeight)) {
-        avatar = QImage(":/images/account-36.png");
+        avatar = QImage(":/images/account.png");
     }
+
+    QRect actualRect(0, 0, kAvatarWidth * devicePixelRatio() , kAvatarHeight * devicePixelRatio());
+    avatar.scaled(actualRect.size());
+    QImage masked_image(actualRect.size(), QImage::Format_ARGB32_Premultiplied);
+    masked_image.fill(Qt::transparent);
+    QPainter mask_painter;
+    mask_painter.begin(&masked_image);
+    mask_painter.setRenderHint(QPainter::Antialiasing);
+    mask_painter.setCompositionMode(QPainter::CompositionMode_SourceOver);
+    mask_painter.setBrush(Qt::white);
+    mask_painter.drawEllipse(actualRect);
+    mask_painter.setCompositionMode(QPainter::CompositionMode_SourceIn);
+    mask_painter.drawImage(actualRect, avatar);
+    mask_painter.setCompositionMode(QPainter::CompositionMode_DestinationOver);
+    mask_painter.fillRect(actualRect, Qt::transparent);
+    mask_painter.end();
+#if (QT_VERSION >= QT_VERSION_CHECK(5, 0, 0))
```

Comment:
The above code manipulating the mask image is the same as those in `accounts-view.cpp`, can we refactor them out to a common function?